src/components/Tabs/ScatterPlotTab.tsx

```diff
@@ -796,7 +795,7 @@ export default memo(function ScatterPlotTab(props: ScatterPlotTabProps): ReactEl
           label={"X"}
           selected={xAxisFeatureKey || ""}
           items={menuItems}
-          onChange={(key) => props.updateScatterPlotConfig({ xAxis: key })}
+          onChange={(key: string) => props.updateScatterPlotConfig({ xAxis: key })}
```

Comment:
for consistency: should this still be `key`? or should it be changed to `value` as well?